### ISSUE TITLE
lSTM Keras recurrent.py forget gate

### ISSUE THREAD

**Issue by mhzaboli** [keras-team/keras#14283]
I am implementing LSTM in Keras and try to customize it for my own use. Actually, I want to do a change in the number of layers. In keras/layer/recurrent.py I want to delete the sigmoid function from the forget layer in the LSTM cell. Could anyone help me to figure it out urgently?

**Comment by ivallesp** [comment]
Could you please post a reproducible snippet of code so that we can try to help?

**Comment by yash-clear** [comment]
you can modify 'recurrent activation' parameter for deleting sigmoid function from forget layer.
recurrent_activation : Activation function to use for the recurrent step. Default: sigmoid (sigmoid). If you pass None, no activation is applied (ie. "linear" activation: a(x) = x).

**Comment by sushreebarsa** [comment]
@mhzaboli Could you please refer to this [link](https://keras.io/api/layers/recurrent_layers/lstm/) and let us know if it helps ! In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by sushreebarsa** [comment]
Closing this issue due to lack of recent activity & assuming this issue as resolved .Please feel free to reopen if you still have concern.Thanks!
